view/error404.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>404 - Page Not Found</title>
+    <link rel="stylesheet" href="css/error404.css">
```

Comment:
Please reference the full link for the CSS to ensure it’s compatible with the routing `/view/css/error404.css`